### PR TITLE
✨ feat: add LLM powered assistant module

### PR DESCRIPTION
**Pull request by cosmic-flood**
✨ feat: add LLM powered assistant module for Feature Flags & Experimentation service

1. Add LLM folder as a independent experimental feature
2. Add `prompt ChatCompletion` service for "remove dead feature flags"
3. Add fine-tuning backup folder to do continuously research

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@
 **/.DS_Store
 /infra/mongodb/data/db/
 /infra/redis/data/
+benchmark/k6-scripts/test-results/
 
-benchmark/k6-scripts/test-results/
\ No newline at end of file
+
```
Comment (deleteLater) [inline .gitignore]: remove unnecessary blank lines

File: llm/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- `apikey`: Your API key for accessing the ChatGPT API service.
+- `ffKey`: The feature flag you wish to remove from your project.
+- `variation`: The feature flag return value corresponding to the code you want to retain.
+- `codePath`: The file path of the code you want to scan and where you want to remove dead feature flags.
+
+You can run the Python file using the command below:
+
+```shell
+python3 chat-completion-cli.py --apikey "fake-bBGB03xcxUQi891GdeqFakeqvOCeTMsOr9rE0M" --ffKey "language" --variation "en-us" --codePath "/mnt/c/Code/featbit/featbit/llm/dotnet-sample/U1Prompt/Program.cs"
```
Comment (deleteLater) [inline llm/README.md]: prefer `--apikey "your-openapi-key"`

File: .gitignore (hunk 1 above)
Comment (deleteLater) [inline .gitignore]: remove unnecessary blank lines

File: llm/README.md (hunk 2 above)
Comment (cosmic-flood) [reply llm/README.md]: Resolved

File: .gitignore (hunk 1 above)
Comment (cosmic-flood) [reply .gitignore]: Resolved

**Review by deleteLater — APPROVED** [review]
LGTM

**Pull request merged**